CHANGELOG.md

```diff
@@ -19,7 +19,6 @@ This release extends support of the SDK to OneView REST API version 1600 (OneVie
 - Logical Interconnect
 - Logical Interconnect Group
 - OS Deployment Plan
-- Server Certificate
```

Comment:
Not sure why Server Certificate was removed, 
Need to add Scope in the list.

Reply:
Resolved Merge conflicts